lib/ts/framework/utils.ts

```diff
@@ -137,16 +137,22 @@ export async function assertThatBodyParserHasBeenUsedForExpressLikeRequest(
             let err = await new Promise((resolve) => {
                 let resolvedCalled = false;
                 /**
-                 * the setImmediate here is to counter the next.js issue
-                 * where the json parser would not resolve and thus the request
-                 * just hangs forever. Next.JS does json parsing on its own.
+                 * Nextjs allow users to disable the default parser.
+                 * To handle that scenario, we are still parsing the request body
                  */
```

Comment:
Need to test that if user has disabled bodyParser in NextJS, our parser runs and it doesn't get "stuck"